libkbfs/md_journal_test.go

```diff
@@ -585,6 +614,8 @@ func testMDJournalBranchConversion(t *testing.T, ver MetadataVer) {
 	mdcache := NewMDCacheStandard(10)
 	cachedMd := makeMDForTest(
 		t, ver, id, firstRevision, j.uid, signer, firstPrevRoot)
+	err := cachedMd.bareMd.SignWriterMetadataInternally(ctx, codec, signer)
```

Comment:
What's the point of moving this out of `makeMDForTest`?  Shouldn't a true mdv2 benchmark include the signature?